nih-sparc/sparc-app
# Find data page - initial functionality #44bvwu

# Description

The purpose of this PR is to add the initial functionality for the find data page.
This includes:
- Basic contentful API search integration for Projects and Help
- Basic layout for sidebar, tables, and tabs. Each table is its own search results.

One gotcha I noticed when working with this was with `asyncData` and HMR. When HMR, any data brought in with `asyncData` won't change, which is an issue here because that data needs to change when switching search type tabs.

## Type of change


# How Has This Been Tested?

- Go to the Find Data page by using the main navigation
- The first search type tab should be active, and it should be showing you results.
- Use the search input in the hero to make searches
- Click on the "Projects" tab. Projects should load.

# Checklist:
